marlowe/doc/playground-blockly.adoc

```diff
@@ -3,10 +3,79 @@
 
 == Marlowe in Blockly
 
-So far in these tutorials we have concentrated on building contracts in the textual version of Marlowe, embedded in Haskell, Marlowe contracts can also be built using the Blockly visual programming environment, as described in this series of videos.
+So far in these tutorials we have concentrated on building contracts in the textual version of Marlowe, embedded in Haskell, Marlowe contracts can also be built using the Blockly visual programming environment, as we describe here.
 
+=== Getting started
+
+To start building a Blockly project in the Marlowe Playground, which you can find out more about in the section about xref:playground-overview#playground-overview[], select *New Project* and then *Blockly* for the project type. You'll now see a window like this:
```

Comment:
I wouldn't say `window`, since there is no window in the image

Reply:
used screen instead